users/discord/components/UsersSection.js

```diff
@@ -18,7 +54,10 @@ export const UsersSection = ({ users, showUser, handleUserSelected }) => {
             src: user?.picture?.url ?? dummyPicture,
             class: 'user_image',
           }),
-          createElement('span', {}, [user.first_name + ' ' + user.last_name]),
+          // createElement('span', {}, [
+          //   user.first_name + ' ' + user.last_name + user.username,
+          // ]),
```

Comment:
can we get rid of this if not needed?

Reply:
Same, this is being currently used with the actual data I have just replicated my version to test out the mock data.